StarkInfra/StarkInfra/PixClaim/PixClaim.cs

```diff
@@ -213,9 +224,10 @@ public static PixClaim Get(string id, User user = null)
         ///     <item>status [list of strings, default null]: filter for status of retrieved objects. Options: new List<string>{ "created", "failed", "delivered", "confirmed", "success", "canceled" }</item>
         ///     <item>ids [list of strings, default null]: list of ids to filter retrieved objects. ex: new List<string>{ "5656565656565656", "4545454545454545" }</item>
         ///     <item>type [strings, default null]: filter for the type of retrieved PixClaims. Options: "ownership" or "portability".</item>
-        ///     <item>agent [string, default null]: filter for the agent of retrieved PixClaims. Options: "claimer" or "claimed".</item>
         ///     <item>keyType [string, default null]: filter for the PixKey type of retrieved PixClaims. Options: "cpf", "cnpj", "phone", "email" and "evp",</item>
-        ///     <item>keyId [string, default null]: filter PixClaims linked to a specific PixKey id. Example: "+5511989898989".</item>
+        ///     <item>keyID [string, default null]: filter PixClaims linked to a specific PixKey id. Example: "+5511989898989".</item>
+        ///     <item>flow [string, default null]: filter for the flow of retrieved PixClaims. Options: "claimer" or "claimed".</item>
```

Comment:
`direction of the Pix Claim. Options: "in" if you received the PixClaim or "out" if you created the PixClaim.`